dapps/marketplace/src/pages/listing/_ListingPending.js

```diff
@@ -2,20 +2,32 @@ import React from 'react'
 import { fbt } from 'fbt-runtime'
 
 import Link from 'components/Link'
+import Price from 'components/Price'
 
-const Pending = () => (
-  <div className="listing-buy pending">
-    <fbt desc="listingPending.text">
-      <div>This listing is</div>
-      <div>Pending</div>
-      <div>
-        Another buyer has already made an offer on this listing. Try visiting
-        the listings page and searching for something similar.
+const Pending = ({ listing, isMobile }) => (
+  <div className="listing-buy">
+    <div className="price">
+      <Price listing={listing} descriptor />
+    </div>
+    <div className="status">
+      <div className="status-title">
+        <fbt desc="Pending">Pending</fbt>
       </div>
-    </fbt>
-    <Link to="/listings">
-      <fbt desc="viewListings">View Listings</fbt>
-    </Link>
+      <div className="status-text">
+        {!isMobile
+          ? fbt(
```

Comment:
I think we should try to use the `<fbt>` component when possible

Reply:
Have changed it. Just want to know if there is any reason to prefer `<fbt>` over `fbt()`?

Reply:
I think it's easier to read when the `desc=` param is explicit